cmake/modules/hunter_get_lang_standard_flag.cmake

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) 2017 Pawel Bylica
+# All rights reserved.
+
+include(hunter_status_debug)
+
+function(hunter_get_lang_standard_flag LANG OUTPUT)
+  set(CXX_standards 17 14 11 98)
+  set(C_standards 11 99 90)
+  # Find the <lang> standard flag.
+  # This maps the logic in the CMake code:
+  # https://github.com/Kitware/CMake/blob/3bccdd89c88864839a0c8d4ea56bd069c90fa02b/Source/cmLocalGenerator.cxx#L1433-L1467
+
+  hunter_status_debug("CMAKE_${LANG}_STANDARD_DEFAULT: ${CMAKE_${LANG}_STANDARD_DEFAULT}")
+  hunter_status_debug("CMAKE_${LANG}_STANDARD: ${CMAKE_${LANG}_STANDARD}")
+  hunter_status_debug("CMAKE_${LANG}_EXTENSIONS: ${CMAKE_${LANG}_EXTENSIONS}")
+  hunter_status_debug("CMAKE_${LANG}_STANDARD_REQUIRED: ${CMAKE_${LANG}_STANDARD_REQUIRED}")
+
+  set("${OUTPUT}" "" PARENT_SCOPE)  # Reset output in case of quick return.
+
+  string(COMPARE EQUAL "${CMAKE_${LANG}_STANDARD_DEFAULT}" "" no_default)
+  if(no_default)
+    # This compiler has no notion of language standard levels.
```

Comment:
Please add this link as a comment:
* https://github.com/Kitware/CMake/blob/3bccdd89c88864839a0c8d4ea56bd069c90fa02b/Source/cmLocalGenerator.cxx#L1427-L1432